mbed_project/_internal/project_data.py

```diff
@@ -103,7 +103,7 @@ def from_new(cls, root_path: Path) -> "MbedProgramData":
 
         mbed_file.touch()
         config.touch()
-        mbed_os_ref.write_text(MBED_OS_REFERENCE_URL)
+        mbed_os_ref.write_text(f"{MBED_OS_REFERENCE_URL}#master")
```

Comment:
Thanks for making this explicit. Does this mean that when a new project is created that head of the master branch of Mbed OS is used?

Reply:
Yes, exactly; for the moment it will check out the master branch. I'm not sure whether we actually want to pin it to the latest mbed-os release? If it were the case that we wanted to pin to the latest released version, it might be nice in the future to have some way of updating this automatically when a new mbed-os update drops.